src/main/java/com/cburch/logisim/std/arith/FPToInt.java

```diff
@@ -81,7 +81,7 @@ public FPToInt() {
         new Object[] {BitWidth.create(8), BitWidth.create(32), ROUND_OPTION});
     setKeyConfigurator(new BitWidthConfigurator(StdAttr.FP_WIDTH));
     setOffsetBounds(Bounds.create(-40, -20, 40, 40));
-    setIcon(new ArithmeticIcon("FP→I", 2));
+    setIcon(new ArithmeticIcon("FP\u2192I", 2));
```

Comment:
Please add comment explaining why using `\u` is required here. Or, maybe, also add a issue ticket (low pri) and link to it from code comment.

Reply:
I replaced inline Unicode right arrow characters in the Java source code with the `\uNNNN` syntax.

The `\uNNNN` syntax is the correct way to use Unicode characters in Java source code and there are many instances of this in the logisim source code, including other Arithmetic components, and `\u2192` is already used for a right arrow character in the Java class `ShiftRegister` and `PlaRomPanel`.